src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -357,21 +357,48 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
 
   render() {
     const { artwork, inquiryComponent } = this.props
+    const isPriceListed = !artwork?.isPriceHidden
+    const isOfferableFromInquiry = artwork?.isOfferableFromInquiry
+    const isOfferable = artwork?.is_offerable
+    const isAcquireable = artwork?.is_acquireable
+    const isInquireable = artwork?.is_inquireable
 
     const {
       isCommittingCreateOrderMutation,
       isCommittingCreateOfferOrderMutation,
       selectedEditionSet,
     } = this.state
 
-    const artworkEcommerceAvailable = !!(
-      artwork.is_acquireable || artwork.is_offerable
-    )
+    const artworkEcommerceAvailable = !!(isAcquireable || isOfferable)
 
-    if (!artwork.sale_message && !artwork.is_inquireable) {
+    if (!artwork.sale_message && !isInquireable) {
       return <Separator />
     }
 
+    const shouldDisplayMakeOfferButton = (): boolean => {
+      if (isOfferable || (isPriceListed && isOfferableFromInquiry)) {
+        return true
+      }
+      return false
+    }
+
+    const shouldDisplayMakeOfferAsPrimary = (): boolean => {
+      if (
+        shouldDisplayMakeOfferButton() &&
+        shouldDisplayContactGalleryButton()
+      ) {
+        return true
+      }
+      return false
+    }
+
+    const shouldDisplayContactGalleryButton = (): boolean => {
+      if (isInquireable && !isAcquireable && !isOfferable) {
+        return true
+      }
+      return false
+    }
```

Comment:
And same here, about directly returning the condition.

Reply:
Ah I see what you mean. This was just a refactor to [clean this up ](https://github.com/artsy/force/blob/a1d6c27a6b8a29eaeb9778c19cd4686e95d3e7d3/src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx#L459)a bit.

Reply:
@ashleyjelks seems that `isInquireable` needs to be checked as boolean like the other two:
```
const shouldDisplayContactGalleryButton = (): boolean => !!isInquireable && !isAcquireable && !isOfferable
```